packages/extension-chakra-storefront/config/default.json

```diff
@@ -0,0 +1,120 @@
+{
+    "enabled": true,
+    "activeDataEnabled": false,
+    "commerceAPI": {
+        "proxyPath": "/mobify/proxy/api",
+        "parameters": {
+            "clientId": "<CLIENT_ID>",
+            "organizationId": "<ORGANIZATION_ID>",
+            "shortCode": "<SHORT_CODE>",
+            "siteId": "<SITE_ID>"
+        }
+    },
+    "defaultSite": "<SITE_ID>",
+    "einsteinAPI": {
+        "host": "https://api.cquotient.com",
+        "einsteinId": "<EINSTEIN_ID>",
+        "siteId": "<EINSTEIN_SITE_ID>",
+        "isProduction": false
+    },
+    "pages": {
+        "Account": true,
+        "Cart": true,
+        "Checkout": true,
+        "CheckoutConfirmation": true,
+        "Home": true,
+        "Login": true,
+        "Registration": true,
+        "ResetPassword": true,
+        "LoginRedirect": true,
+        "ProductDetail": true,
+        "ProductList": true,
+        "Wishlist": true
+    },
+    "routes": [
```

Comment:
I think having both routes and pages might be a little confusing. I know you had asked me what my intentions were with the pages configuration, but I only remembered now or thought of this idea. 

The pages type could be a map where the value is boolean or a route configuration excluding the component. 

What do you think?

Reply:
Note: we decided to remove this `routes` and repurpose `pages` such that it accepts both boolean and the path string.